site/src/pages/TemplatePage/TemplateEmbedPage/TemplateEmbedPage.tsx

```diff
@@ -126,6 +153,40 @@ export const TemplateEmbedPageView: FC<TemplateEmbedPageViewProps> = ({
 								</RadioGroup>
 							</FormSection>
 
+							<div className="flex flex-col gap-1">
+								<Label className="text-md" htmlFor={defaultWorkspaceNameID}>
+									Workspace name
+								</Label>
+								<div
+									className={"text-sm mb-3"}
+									css={(theme) => ({
+										color: theme.palette.text.secondary,
+									})}
+								>
+									Default name for the new workspace
+								</div>
+								<Input
+									id={defaultWorkspaceNameID}
+									value={buttonValues.name}
+									onChange={(event) => {
+										validateWorkspaceName(event.target.value);
+										setButtonValues((buttonValues) => ({
+											...buttonValues,
+											name: event.target.value,
+										}));
+									}}
+								/>
+								<div
+									className="text-sm mt-1"
+									role="alert"
+									css={(theme) => ({
+										color: theme.palette.error.main,
+									})}
+								>
```

Comment:
we should use a tailwind class here instead of emotion. I'm not sure what color to recommend tho, maybe `text-highlight-red`? I don't see this in any of your screenshots.

it'd also be nice to have a story for this

Reply:
Switched to `text-highlight-red` and added storybook